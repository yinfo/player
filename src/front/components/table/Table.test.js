// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Table from './Table';

describe('Table component', () => {
  it('renders as expected', () => {
    const component = shallow(<Table>{<p>a child</p>}</Table>);
    expect(component).toMatchSnapshot();
  });
});
