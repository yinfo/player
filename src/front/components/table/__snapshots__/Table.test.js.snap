// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table component renders as expected 1`] = `ShallowWrapper {}`;
